packages/firestore/src/local/local_store.ts

```diff
@@ -1249,9 +1249,28 @@ export async function ignoreIfPrimaryLeaseLoss(
   }
 }
 
-export interface ApplyBundleDocumentsResult {
-  changedDocuments: MaybeDocumentMap;
-  queryDocumentMap: Map<string, DocumentKeySet>;
+export function getQueryDocumentMapping(
+  localStore: LocalStore,
+  documents: BundledDocuments
+): Map<string, DocumentKeySet> {
+  const localStoreImpl = debugCast(localStore, LocalStoreImpl);
+  const queryDocumentMap = new Map<string, DocumentKeySet>();
+  const bundleConverter = new BundleConverter(localStoreImpl.serializer);
```

Comment:
Is it possible to get the serializer from the callsite of this function, drop its dependency on "LocalStore" and move it to bundle.ts?

Reply:
I had to add a `getSerializer` to localstore interface, but that still seems better than current.